utils/utils.py

```diff
@@ -376,3 +377,11 @@ async def get_file_from_url(
     return discord.File(
         BytesIO(response), filename=filename
     )  # convert response to files
+
+
+def is_user_banned_openAI(user: Identifiable, /):
+    """returns true if user is banned from openAI commands"""
+    db_user = get_database_user(user)
+    if not db_user:
+        return False
+    return db_session.query(OpenAI).filter(OpenAI.user_id == db_user.id).count() == 1
```

Comment:
I would prefer `.first() is not None` - since sqla caches results, so the user object should be kicking around somewhere, but not sure if that extends to count.